src/main/java/se/sundsvall/webmessagecollector/api/MessageResource.java

```diff
@@ -29,52 +30,55 @@
 @Tag(name = "messages", description = "Messages")
 @ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class})))
 @ApiResponse(responseCode = "404", description = "Not found", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 @ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
-public class MessageResource {
+class MessageResource {
 
 	private final MessageService service;
 
-	public MessageResource(final MessageService service) {
+	MessageResource(final MessageService service) {
 		this.service = service;
 	}
 
 	@GetMapping(value = "/{familyId}/{Instance}", produces = APPLICATION_JSON_VALUE)
 	@ApiResponse(responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true)
 	@Operation(summary = "Get a list of messages related to a specific familyId", description = "Returns a list of messages found for the specified familyId")
-	public ResponseEntity<List<MessageDTO>> getMessages(
-		@Parameter(name = "familyId", description = "FamilyId to fetch messages for", example = "123", required = true) @PathVariable("familyId") final String familyId,
-		@Parameter(name = "Instance", description = "Instance to fetch messages for", example = "INTERNAL", required = true) @PathVariable("Instance") final String instance) {
-		return ResponseEntity.ok(service.getMessages(familyId, instance));
+	ResponseEntity<List<MessageDTO>> getMessages(
+			@Parameter(name = "municipalityId", description = "Municipality Id", example = "2281", required = true) @PathVariable("municipalityId") @ValidMunicipalityId final String municipalityId,
+			@Parameter(name = "familyId", description = "FamilyId to fetch messages for", example = "123", required = true) @PathVariable("familyId") final String familyId,
+			@Parameter(name = "Instance", description = "Instance to fetch messages for", example = "INTERNAL", required = true) @PathVariable("Instance") final String instance) {
+		return ResponseEntity.ok(service.getMessages(municipalityId, familyId, instance));
 	}
 
 	@DeleteMapping
 	@ApiResponse(responseCode = "204", description = "No Content", useReturnTypeSchema = true)
 	@Operation(summary = "Delete a list of messages", description = "Deletes a list of messages with the ids provided")
-	public ResponseEntity<Void> deleteMessages(@RequestBody final List<Integer> ids) {
+	ResponseEntity<Void> deleteMessages(
+			@Parameter(name = "municipalityId", description = "Municipality Id", example = "2281", required = true) @PathVariable("municipalityId") @ValidMunicipalityId final String municipalityId,
+			@RequestBody final List<Integer> ids) {
 		service.deleteMessages(ids);
 		return ResponseEntity.noContent().build();
```

Comment:
Add this in order to mitigate "application/octet-stream"-content-type that WSO2-will add otherwise?
`.header(CONTENT_TYPE, ALL_VALUE)`